tfhe/benches/integer/bench.rs

```diff
@@ -1400,6 +1404,121 @@ mod cuda {
         bench_group.finish()
     }
 
+    fn cuda_compress(c: &mut Criterion) {
+        let bench_name = "integer::cuda::compression";
+        let mut bench_group = c.benchmark_group(bench_name);
+        bench_group
+            .sample_size(15)
+            .measurement_time(std::time::Duration::from_secs(30));
+
+        let stream = CudaStreams::new_multi_gpu();
+
+        let param = PARAM_MESSAGE_2_CARRY_2_KS_PBS;
```

Comment:
those are not the parameters matching the comp_param below, you want the TUniform params as well

Reply:
Can we use `TUNIFORM` with the GPU implementation? We believed  we should be using gaussian.

Reply:
Tuniform is just the distribution of noise at keygen time, no impact for gpu, also the BC is tuniform so it better work 😅

Reply:
Good to know! Fixed.